dev/bots/analyze.dart

```diff
@@ -541,6 +544,54 @@ Future<void> verifyInternationalizations() async {
   }
 }
 
+Future<void> verifyNoRuntimeTypeInToString(String workingDirectory) async {
+  final List<File> files = _allFiles(path.join(workingDirectory, 'packages', 'flutter', 'lib'), 'dart', minimumMatches: 400)
+      .where((File file) => !file.path.endsWith(path.join('lib', 'src', 'foundation', 'object.dart')))
```

Comment:
I would hoist this into a kExcludedFiles or similar

Reply:
Done